field/src/batch_inverse.rs

```diff
@@ -24,22 +45,10 @@ pub fn batch_multiplicative_inverse<F: Field>(x: &[F]) -> Vec<F> {
     // Handle special cases. Paradoxically, below is repetitive but concise.
     // The branches should be very predictable.
```

Comment:
I think these comments are now broken as you deleted what they refer to.

Reply:
Fixed